src/models/hashmodel.h

```diff
@@ -104,6 +106,16 @@ class HashModel : public QAbstractTableModel
         return index(row, column);
     }
 
+    QModelIndex indexForSymbol(Data::Symbol symbol)
```

Comment:
sorry, I mean `SymbolCostModelImpl`, and yes make m_rows protected for that purpose